base/server/python/pki/server/__init__.py

```diff
@@ -719,6 +719,13 @@ def create(self, force=False):
                 host_dir = os.path.join(engine_dir, host_name)
                 self.makedirs(host_dir, force=force)
 
+                # symlink the rewrite.config in the host dir
+                target = os.path.join(
+                    PKIServer.SHARE_DIR, 'server', 'conf',
+                    'Catalina', 'localhost', 'rewrite.config')
+                link = os.path.join(host_dir, 'rewrite.config')
+                self.symlink(target, link, force=force)
+
```

Comment:
I am installing last version of EST before to modify the realm and this step did not work. The rewrite.config was not set-up with the `create`. I will do some other test after the realm to verify if I did a mistake or there is a problem

Reply:
I have done the additional tests and it worked properly!